src/extension.ts

```diff
@@ -37,6 +89,7 @@ export async function activate(context: vscode.ExtensionContext) {
         e.affectsConfiguration("gitdoc.autoCommitDelay") ||
         e.affectsConfiguration("gitdoc.filePattern")) {
         checkEnabled(git);
+        debouncedUpdateStatusBar(vscode.window.activeTextEditor);
```

Comment:
Similar to my comment above, I think we should update the status bar in the checkEnabled function, and simplify this code.